sdk/java/src/ai/picovoice/picovoice/PicovoiceException.java

```diff
@@ -13,6 +13,10 @@
 package ai.picovoice.picovoice;
 
 public class PicovoiceException extends Exception {
+    PicovoiceException(String message, Throwable cause) {
```

Comment:
Where possible in the Picovoice SDKs, we're converting the ppn/rhn exception to a pv exception of the same type. This allows the user to catch based on a single exception type, e.g.
```java
try{}
catch(PicovoiceInvalidArgumentException e) {}
catch(PicovoiceActivationException e) {}
```
This is done in the [Android binding](https://github.com/Picovoice/picovoice/blob/v2.0/sdk/android/Picovoice/picovoice/src/main/java/ai/picovoice/picovoice/Picovoice.java)